apps/www/components/Front/index.js

```diff
@@ -161,6 +134,34 @@ const Front = ({
     url: `${PUBLIC_BASE_URL}${front.meta.path}`,
   }
 
+  const fetchMore = () => {
+    return nativeFetchMore({
+      variables: {
+        first: 15,
+        after: data.front && data.front.children.pageInfo.endCursor,
+        before: undefined,
+      },
+      updateQuery: (previousResult = {}, { fetchMoreResult = {} }) => {
+        const previousSearch = previousResult.front.children || {}
+        const currentSearch = fetchMoreResult.front.children || {}
+        const previousNodes = previousSearch.nodes || []
+        const currentNodes = currentSearch.nodes || []
+        const res = {
+          ...previousResult,
+          front: {
+            ...previousResult.front,
+            children: {
+              ...previousResult.front.children,
+              nodes: [...previousNodes, ...currentNodes],
+              pageInfo: currentSearch.pageInfo,
+            },
+          },
+        }
+        return res
+      },
+    }).catch((error) => console.error(error))
```

Comment:
if we do not catch it our report error with catch it as uncaught exception
```suggestion
    })
```